src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/StringBasedOpt.java

```diff
@@ -105,4 +115,51 @@ public String getTranslationStatus() {
         }
         return status;
     }
+
+    public void getMultiVersionKeyMessages(MessageCacheItem cacheItem) {
```

Comment:
the function name should be a string like 'updateKeyCacheByVersion', or 'refreshKeyCacheByVersion'?